modules/apps/frontend-js/frontend-js-aui-web/npmscripts.config.js

```diff
@@ -13,7 +13,9 @@
  */
 
 module.exports = {
-	bundling: false,
+	build: {
+		bundler: false,
+	},
```

Comment:
Can you remove the `federation` stuff at the same time?

Reply:
Mmm, no, because we have the [other PR](https://github.com/liferay-frontend/liferay-portal/pull/938) waiting and I prefer:

1) Not to add too many changes as we are near the release (:roll_eyes: )
2) Minimize the conflicts with the other PR as I'm sure that when I retake it it's going to give some headache :-(

Reply:
You've already lost the battle on point "2" because you're touching the lines in this PR that are directly adjacent to lines you want to touch in the other PR.

As for "1", how can this possibly be risky? This is literally a dead config property in the new version of `@liferay/npm-scripts`, and by not removing it, you are distributing a change that could be atomic over two PRs.

Up to you though. The PR looks good to me otherwise.

Reply:
Ah, you mean removing it for this modified project only? If it is only for this, then it's OK. I'll remove it since CI has failed.

I thought you were talking about cleaning all federation configurations... :scream: